### DISCUSSION TITLE
I can't get the rtsp stream

### DISCUSSION THREAD

**Discussion by volkankoc** [bluenviron/mediamtx#1576] [Questions]
my yml file
```
paths:
  test:
    source: rtsp://admin:admin!@192.168.1.211:554/cam/realmonitor?channel=1&subtype=1
```
vlc player test url this but not working
http://localhost:8888/test

error code:
no one is publishing to path 'test'

**Comment by JeDiE99** [comment]
where server logs ?

**Reply by volkankoc** [reply]
> where server logs ?

log info

`2023/03/16 22:31:21 INF MediaMTX / rtsp-simple-server v0.21.6
2023/03/16 22:31:21 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/03/16 22:31:21 INF [RTMP] listener opened on :1935
2023/03/16 22:31:21 INF [HLS] listener opened on :8888
2023/03/16 22:31:21 INF [WebRTC] listener opened on :8889 (HTTP)
2023/03/16 22:31:21 INF [API] listener opened on 127.0.0.1:9997
2023/03/16 22:31:33 INF [HLS] [muxer kanal1] created (requested by ::1)
2023/03/16 22:31:33 INF [HLS] [muxer kanal1] destroyed (no one is publishing to path 'kanal1')
2023/03/16 22:31:33 INF [HLS] [muxer kanal1] created (requested by ::1)
2023/03/16 22:31:33 INF [HLS] [muxer kanal1] destroyed (no one is publishing to path 'kanal1')
2023/03/16 22:31:42 INF shutting down gracefully
2023/03/16 22:31:42 INF [API] listener is closing
2023/03/16 22:31:42 INF [RTSP] listener is closing
2023/03/16 22:31:42 INF [WebRTC] listener is closing
2023/03/16 22:31:42 INF [HLS] listener is closing
2023/03/16 22:31:42 INF [RTMP] listener is closing
2023/03/16 22:31:42 INF waiting for external commands`

log debug
`2023/03/16 22:30:09 INF shutting down gracefully
2023/03/16 22:30:09 DEB [path kanal1] destroyed (terminated)
2023/03/16 22:30:09 INF [API] listener is closing
2023/03/16 22:30:09 INF [RTSP] listener is closing
2023/03/16 22:30:09 DEB path manager is shutting down
2023/03/16 22:30:09 INF [WebRTC] listener is closing
2023/03/16 22:30:09 INF [HLS] listener is closing
2023/03/16 22:30:09 INF [RTMP] listener is closing
2023/03/16 22:30:09 INF waiting for external commands
2023/03/16 22:30:12 INF MediaMTX / rtsp-simple-server v0.21.6
2023/03/16 22:30:12 DEB [path kanal1] created
2023/03/16 22:30:12 DEB path manager created
2023/03/16 22:30:12 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/03/16 22:30:12 INF [RTMP] listener opened on :1935
2023/03/16 22:30:12 INF [HLS] listener opened on :8888
2023/03/16 22:30:12 INF [WebRTC] listener opened on :8889 (HTTP)
2023/03/16 22:30:12 INF [API] listener opened on 127.0.0.1:9997
2023/03/16 22:30:37 DEB [HLS] [conn ::1] GET /kanal1
2023/03/16 22:30:37 DEB [HLS] [conn ::1] [c->s] GET /kanal1 HTTP/1.1
Host: localhost:8888
Accept: */*
Accept-Language: tr
Range: bytes=0-
User-Agent: VLC/3.0.17.4 LibVLC/3.0.17.4


2023/03/16 22:30:37 DEB [HLS] [conn ::1] [s->c] HTTP/1.1 301 Moved Permanently
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Location: /kanal1/
Server: rtsp-simple-server


2023/03/16 22:30:37 DEB [HLS] [conn ::1] GET /kanal1/
2023/03/16 22:30:37 DEB [HLS] [conn ::1] [c->s] GET /kanal1/ HTTP/1.1
Host: localhost:8888
Accept: */*
Accept-Language: tr
Range: bytes=0-
User-Agent: VLC/3.0.17.4 LibVLC/3.0.17.4


2023/03/16 22:30:37 INF [HLS] [muxer kanal1] created (requested by ::1)
2023/03/16 22:30:37 INF [HLS] [muxer kanal1] destroyed (no one is publishing to path 'kanal1')
2023/03/16 22:30:37 DEB [HLS] [conn ::1] [s->c] HTTP/1.1 404 Not Found
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Server: rtsp-simple-server


2023/03/16 22:30:37 DEB [HLS] [conn ::1] GET /kanal1/
2023/03/16 22:30:37 DEB [HLS] [conn ::1] [c->s] GET /kanal1/ HTTP/1.0
Host: localhost:8888
Connection: close
Icy-Metadata: 1
User-Agent: VLC/3.0.17.4 LibVLC/3.0.17.4


2023/03/16 22:30:37 INF [HLS] [muxer kanal1] created (requested by ::1)
2023/03/16 22:30:37 INF [HLS] [muxer kanal1] destroyed (no one is publishing to path 'kanal1')
2023/03/16 22:30:37 DEB [HLS] [conn ::1] [s->c] HTTP/1.1 404 Not Found
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Server: rtsp-simple-server


2023/03/16 22:30:43 INF shutting down gracefully
2023/03/16 22:30:43 DEB [path kanal1] destroyed (terminated)
2023/03/16 22:30:43 INF [API] listener is closing
2023/03/16 22:30:43 INF [RTSP] listener is closing
2023/03/16 22:30:43 DEB path manager is shutting down
2023/03/16 22:30:43 INF [WebRTC] listener is closing
2023/03/16 22:30:43 INF [HLS] listener is closing
2023/03/16 22:30:43 INF [RTMP] listener is closing
2023/03/16 22:30:43 INF waiting for external commands
`

**Comment by volkankoc** [comment]
I try this config but not working

`paths:
  kanal1:
    source: rtsp://wowzaec2demo.streamlock.net/vod/mp4:BigBuckBunny_115k.mp4`

**Comment by JeDiE99** [comment]
By logs no info about initialization kanal1 by server on start -> check your config

**Comment by volkankoc** [comment]
I got it working but I have a few questions.
This link works in vlc player
rtsp://localhost:8554/channel1
on the web
  <iframe src="http://localhost:8889/channel1" scrolling="no" width="800" height="600"></iframe>
it worked this way.
I want to run it with clappr .js or video.js.
my yml file
[rtsp-simple-server.yml.txt](https://github.com/aler9/rtsp-simple-server/files/11006167/rtsp-simple-server.yml.txt)

**Comment by JeDiE99** [comment]
You need to research source code and adapt for you purpouse, if you don't want to use iframe. Or use hls with hlsjs

**Reply by volkankoc** [reply]
> You need to research source code and adapt for you purpouse, if you don't want to use iframe. Or use hls with hlsjs

Solve this problem by using hls.
Now I cannot watch this m3u8 stream that I have encrypted. I couldn't find the parameters how to monitor.
I created the secret files. Broadcast is encrypted but I can't watch

[a comment by an automated account is not shown]
